packages/ckeditor5-core/src/plugincollection.js

```diff
@@ -165,10 +165,12 @@ export default class PluginCollection {
 	 * or {@link module:core/plugin~PluginInterface.pluginName plugin names}.
 	 * @param {Array.<String|Function>} [removedPlugins] Names of the plugins or plugin constructors
 	 * that should not be loaded (despite being specified in the `plugins` array).
+	 * @param {Array.<Function>} pluginsToReplace An array of {@link module:core/plugin~PluginInterface plugin constructors}
```

Comment:
```suggestion
	 * @param {Array.<Function>} pluginsSubstitutions An array of {@link module:core/plugin~PluginInterface plugin constructors}
```

Reply:
Also, I'd rename `removedPlugins` to `pluginsToRemove`.

Reply:
I'm for `pluginsSubstitutions` as here we're not passing the plugins to be replaced but their replacements and `pluginReplacements` sounds wrong. 

But on the funny note – these words are causing some issues: https://queens-english-society.org/substitute-and-replace/